cmd/swarmctl/node/common.go

```diff
@@ -138,6 +138,16 @@ func getNode(ctx context.Context, c api.ControlClient, input string) (*api.Node,
 		}
 
 		if l := len(rl.Nodes); l > 1 {
+			for _, n := range rl.Nodes {
+				name := n.Spec.Annotations.Name
```

Comment:
@aaronlehmann Right. The Docker doesn't check Annotations.Name. I agree we should keep them consistent.

@stevvooe What do you mean? Do you agree that the client should use n.Description.Hostname directly for displaying node's name?